docs/source/details/backendconfig.rst

```diff
@@ -33,6 +33,18 @@ For a consistent user interface, backends shall follow the following rules:
   Backends should define clearly which keys are applicable to datasets and which are not.
 
 
+Backend-independent JSON configuration
+--------------------------------------
+
+The key ``defer_iteration_parsing`` can be used to optimize the process of opening an openPMD Series.
+By default, a Series is parsed eagerly, i.e. opening a Series implies reading all available iterations.
+Especially when a Series has many iterations, this can be a costly operation and users may wish to defer parsing of iterations to a later point adding ``{"defer_iteration_parsing": true}`` to their JSON configuration.
+
+When parsing non-eagerly, each iteration needs to be explicitly opened with ``Iteration::open()`` before accessing.
+(Notice that Iteration::open() is generally recommended to be used in parallel contexts to avoid parallel file accessing hazards).
```

Comment:
```suggestion
(Notice that ``Iteration::open()`` is generally recommended to be used in parallel contexts to avoid parallel file accessing hazards).
```